.dialyzer_ignore

```diff
@@ -9,3 +9,4 @@ Unknown function 'Elixir.Litmus.Type.PID':'__impl__'/1
 Unknown function 'Elixir.Litmus.Type.Port':'__impl__'/1
 Unknown function 'Elixir.Litmus.Type.Reference':'__impl__'/1
 Unknown function 'Elixir.Litmus.Type.Tuple':'__impl__'/1
+Call to missing or unexported function 'Elixir.Litmus.Type.List':'__impl__'/1
```

Comment:
Dialyzer doesn't throw an error if we change the name of the `List` module. That's why adding the warning here instead of changing the module name.